nservicebus/extending/features.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Feature-based extensibility system
+summary: Explains the concept of feature
+tags:
+ - feature
+ - extensibility
+ - extending
+---
+
+NServiceBus code base consists of two parts.
+
+### Bootstrapping code
+
+The bootstrapping code is responsible for taking the configuration values from the user (via the `BusConfiguration`) API and constructing and starting a bus. The bus is constructed by activating features.
+
+### Message processing code
+
+All the actual logic of the bus is implemented in form of features. Features take part in configuring the underlying dependency injection container with all the services required by the bus to be operational.
+
+## Definition
+
+A feature is defined in a class inheriting from `Feature`. This is a minimal feature that does nothing
+
+<!-- import MinimalFeature -->
+
+
+## Dependencies
+
+Features are so powerful because they can depend on each other to form more complex behaviours.
```

Comment:
remove `are so powerful because they`. either this is obvious from the statement or it isnt. in the first case it is redundant, in the second it seems we are exaggerating.